src/trace/listener.ts

```diff
@@ -70,6 +71,12 @@ export interface TraceConfig {
   coldStartTraceSkipLib: string;
 }
 
+interface SpanPointerAttributes {
+  pointerKind: string;
+  pointerDirection: string;
+  pointerHash: string;
+}
```

Comment:
This interface is defined in two different files. Is there a way we can import it from one to the other?